admin/templates/configuration/tmp/configure/sql/QueryMetadata.sql

```diff
@@ -116,8 +130,12 @@ CREATE OR REPLACE
     `QInfo`.`completed` `Completed`,
     `QInfo`.`returned` `Returned`,
     `QInfo`.`czarId` `CzarId`,
-    REPLACE(`QInfo`.`resultLocation`, '#QID#',  `QInfo`.`queryId`) `ResultLocation`
+    REPLACE(`QInfo`.`resultLocation`, '#QID#',  `QInfo`.`queryId`) `ResultLocation`,
+    `QStatsTmp`.`totalChunks` `totalChunks`,
+    `QStatsTmp`.`completedChunks` `completedChunks`,
+    `QStatsTmp`.`lastUpdate` `lastUpdate`
```

Comment:
Capitalize aliases for three new columns for consistency with other columns? Same for other view.